Makefile

```diff
@@ -147,12 +145,3 @@ integration-tests-container: cmd/wksctl/wksctl cmd/controller/.uptodate
 FORCE:
 
 
-build-docs:
-	@cd docs && docker build -t wksctl-docs .
-
-test-docs: build-docs
-	@docker run -it wksctl-docs /usr/bin/linkchecker _build/html/index.html
-
-serve-docs: build-docs
-	@echo Stating docs website on http://localhost:${DOCS_PORT}/_build/html/index.html
-	@docker run -i -p ${DOCS_PORT}:8000 -e USER_ID=$$UID wksctl-docs
```

Comment:
Is it still possible to run a local docs server now with `mkdocs`? Maybe it'd be good to have some sort of a make script for local testing.

Reply:
Ah yes, I now saw this:

```
pip3 install -r docs/requirements.txt
mkdocs serve
```

So how about putting those under `make serve-docs`? :)